src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -613,7 +613,7 @@ void MarkDynamicDependency (DynamicDependency dynamicDependency, MemberReference
 
 			IEnumerable<IMemberDefinition> members;
 			if (dynamicDependency.MemberSignature is string memberSignature) {
-				members = DocumentationSignatureParser.GetMembersByDocumentationSignature (type, memberSignature);
+				members = DocumentationSignatureParser.GetMembersByDocumentationSignature (type, memberSignature, acceptName: true);
```

Comment:
So this is a light version of wildcard support, right? It will mark any members if `(` is not part of the signature.